src/ecosystem/corals/growth_expanded.jl

```diff
@@ -21,48 +21,47 @@ function growthODE_expanded(du::Array{Float64,2}, X::Array{Float64,2}, p::NamedT
     r = @view p.r[:]  # growth rates
     X_mb = X .* p.mb   # current cover * background mortality
 
-    # 0.3 is the competition rate
-    # Tabular Acropora Enhanced
+    # Aborescent Acropora
```

Comment:
If this has been updated to reflect the changes in this PR, please delete `slow_ODE()`